src/utils/GraphhopperUtils.js

```diff
@@ -143,14 +143,13 @@ function busRoutesAreEqual(busRouteA: Object, busRouteB: Object): boolean {
 function getValidBusRoutes(busRoutes: Array<Object>, busRoutesToCheck: Array<Object>): Array<Object> {
   return busRoutesToCheck.filter((busRouteToCheck) => {
     const isDuplicateRoute = busRoutes.find(busRoute => busRoutesAreEqual(busRouteToCheck, busRoute)) !== undefined;
-    return !isDuplicateRoute;
+    return !isDuplicateRoute && busRouteToCheck.constructor.transfers !== -1;
   });
 }
 
 /**
- * Return { busRoute, walkingRoute } from graphhopper given the parameters
- * walkingRoute contains an array with length 1 containing the shortest possible walking path
- * busRoute contains an array of length 5 with possible paths
+ * Return {routes} from graphhopper given the parameters
+ * routes contains an array of length 5 with possible bus and walking routes
```

Comment:
`{routes} contains an array of up to length 5, with possible bus and walking routes`